pkg/install/generate.go

```diff
@@ -294,6 +309,12 @@ func InstallerPodSpec(
 	}
 	cliImage := *cd.Status.CLIImage
 
+	hiveArg := fmt.Sprintf("/usr/bin/hiveutil install-manager --work-dir /output --log-level debug %s %s", cd.Namespace, provisionName)
+	if cd.Spec.Platform.VSphere != nil {
+		// Add vsphere certificates to CA trust.
+		hiveArg = fmt.Sprintf("cp -v %s/.cacert /etc/pki/ca-trust/source/anchors/ && update-ca-trust && %s", vsphereCloudsDir, hiveArg)
```

Comment:
This will only copy the cert file named `.cacert`. It needs to copy *all* of the certs provided in the certs secret.

Reply:
Changed this to `cp -vr`